<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>76</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="sizeLabel">
     <property name="text">
      <string>Size (all axes)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QgsSpinBox" name="mSizeQgsSpinBox">
     <property name="showGroupSeparator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="sizeBandsLabel">
     <property name="text">
      <string>Size (bands)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QgsSpinBox" name="mSizeBandsQgsSpinBox"/>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="sizeRowsLabel">
     <property name="text">
      <string>Size (rows)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QgsSpinBox" name="mSizeRowsQgsSpinBox"/>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="sizeColsLabel">
     <property name="text">
      <string>Size (cols)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QgsSpinBox" name="mSizeColsQgsSpinBox"/>
   </item>
   <item row="1" column="6">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>105</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsSpinBox</class>
   <extends>QSpinBox</extends>
   <header>qgsspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
